x-pack/plugins/search_indices/public/components/indices/details_page.tsx

```diff
@@ -49,33 +50,40 @@ export const SearchIndexDetailsPage = () => {
     isInitialLoading: isMappingsInitialLoading,
   } = useIndexMapping(indexName);
 
-  const SearchIndexDetailsTabs: EuiTabbedContentTab[] = useMemo(() => {
+  const detailsPageTabs: EuiTabbedContentTab[] = useMemo(() => {
     return [
       {
-        id: SearchIndicesDetailsMappingsTabs.DATA,
+        id: SearchIndexDetailsTabs.DATA,
```

Comment:
corrected mistake. This variable holds all the tabs not just specific to mappings